### PR TITLE
Timestamp fixes etc

### PR DESCRIPTION
**Pull request by Dieterbe**
I think something's up with circle.
this replaces #915

### CODE REVIEW

File: api/dataprocessor.go [hunk 1]
```diff
@@ -501,7 +501,10 @@ func (s *Server) getSeriesCachedStore(ctx *requestContext, until uint32) ([]chun
 	logLoad("cassan", ctx.AMKey, ctx.From, ctx.To)
 
 	log.Debug("cache: searching query key %s, from %d, until %d", ctx.AMKey, ctx.From, until)
-	cacheRes := s.Cache.Search(ctx.ctx, ctx.AMKey, ctx.From, until)
+	cacheRes, err := s.Cache.Search(ctx.ctx, ctx.AMKey, ctx.From, until)
+	if err != nil {
+		return iters, err
+	}
```
Comment (replay) [inline api/dataprocessor.go]: this returns before the following `log.Debug()`, but i think we should either move it up so it is between the call to `.Search()` or just remove it completely if it isn't useful.

File: mdata/cache/ccache.go [hunk 2]
```diff
@@ -209,8 +212,8 @@ func (c *CCache) Search(ctx context.Context, metric schema.AMKey, from, until ui
 		Until: until,
 	}
 
-	if from == until {
-		return res
+	if from >= until {
```
Comment (replay) [inline mdata/cache/ccache.go]: actually this could happen before `CCSearchResult` is instantiated

File: store/cassandra/store_cassandra.go [hunk 3]
```diff
@@ -33,12 +33,12 @@ const Month_sec = 60 * 60 * 24 * 28
 const Table_name_format = `metric_%d`
 
 var (
-	errChunkTooSmall  = errors.New("unpossibly small chunk in cassandra")
-	errStartBeforeEnd = errors.New("start must be before end.")
-	errReadQueueFull  = errors.New("the read queue is full")
-	errReadTooOld     = errors.New("the read is too old")
-	errTableNotFound  = errors.New("table for given TTL not found")
-	errCtxCanceled    = errors.New("context canceled")
+	errChunkTooSmall = errors.New("unpossibly small chunk in cassandra")
```
Comment (replay) [inline store/cassandra/store_cassandra.go]: shouldn't that be `impossibly`? is `unpossibly` a word?

Comment (Dieterbe) [reply store/cassandra/store_cassandra.go]: yes. no.

**Pull request closed without merging**